mappgene/cli.py

```diff
@@ -17,6 +17,12 @@ def parse_args(args):
 
     parser.add_argument('--test', action='store_true',
         help='Test using the example inputs.')
+        
+    parser.add_argument('--threads' default=4, 
+        help='Allows the script to operate more efficiently)
```

Comment:
That's totally true, but let's suppose that the user already knows this and alternatively we can say something more informative to our particular case such as "Number of threads used by the fastp filtering step".